### PR TITLE
feat: add 4 icons to react-northstar-icons

### PR DESCRIPTION
**Pull request by layershifter**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

This PR adds 4 icons to `react-icons-northstar`.

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/fluentui/react-icons-northstar/src/components/BreakoutRoomIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { createSvgIcon } from '../utils/createSvgIcon';
+import { iconClassNames } from '../utils/iconClassNames';
+
+export const BreakoutRoomIcon = createSvgIcon({
```
Comment (layershifter) [inline packages/fluentui/react-icons-northstar/src/components/BreakoutRoomIcon.tsx:6]: I am not sure about naming there. FA has `door-open` for similar icon, https://fontawesome.com/icons/door-open?style=solid. Any ideas?

File: packages/fluentui/react-icons-northstar/src/components/BreakoutRoomIcon.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  svg: ({ classes }) => (
+    <svg viewBox="0 0 12 16" fill="none" className={classes.svg} role="presentation">
+      <path
+        className={cx(iconClassNames.filled, classes.filledPart)}
```
Comment (layershifter) [inline packages/fluentui/react-icons-northstar/src/components/BreakoutRoomIcon.tsx:10]: I don't think that this is accurate... Any suggestions?

image

File: packages/fluentui/react-icons-northstar/src/components/CallTransferredIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { createSvgIcon } from '../utils/createSvgIcon';
+import { iconClassNames } from '../utils/iconClassNames';
+
+export const CallTransferredIcon = createSvgIcon({
```
Comment (layershifter) [inline packages/fluentui/react-icons-northstar/src/components/CallTransferredIcon.tsx:6]: image

Does it have correct look?

File: packages/fluentui/react-icons-northstar/src/components/ComputerAudioIcon.tsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { createSvgIcon } from '../utils/createSvgIcon';
+
+export const ComputerAudioIcon = createSvgIcon({
+  svg: ({ classes }) => (
+    <svg viewBox="0 0 32 32" role="presentation" className={classes.svg}>
```
Comment (layershifter) [inline packages/fluentui/react-icons-northstar/src/components/ComputerAudioIcon.tsx:6]: What about filled/outline variations?

File: packages/fluentui/react-icons-northstar/src/components/ComputerNoAudioIcon.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { createSvgIcon } from '../utils/createSvgIcon';
+
+export const ComputerNoAudioIcon = createSvgIcon({
+  svg: ({ classes }) => (
+    <svg viewBox="0 0 32 32" role="presentation" className={classes.svg}>
```
Comment (layershifter) [inline packages/fluentui/react-icons-northstar/src/components/ComputerNoAudioIcon.tsx:6]: What about filled/outline variations?

File: packages/fluentui/react-icons-northstar/src/components/ComputerNoAudioIcon.tsx [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { createSvgIcon } from '../utils/createSvgIcon';
+
+export const ComputerNoAudioIcon = createSvgIcon({
```
Comment (layershifter) [inline packages/fluentui/react-icons-northstar/src/components/ComputerNoAudioIcon.tsx:4]: We have `SpeakerMuteIcon`, should be this `ComputerAudioMuteIcon`?

image

File: packages/fluentui/react-icons-northstar/src/components/DeviceAudioIcon.tsx [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
....95168,0,0,1-1.28906-.73437,1.824,1.824,0,0,1-.52344-1.35937,1.94218,1.94218,0,0,1,.09375-.60156Q6.2028,24.28174,6.28092,24H5.0153a3.41263,3.41263,0,0,1-1.99219-.66406,2.5472,2.5472,0,0,1-.69531-.78125,2.01432,2.01432,0,0,1-.26562-1.02344v-.14844a.73844.73844,0,0,1,.01563-.14844l1-12.5625A2.67871,2.67871,0,0,1,3.38248,7.625a3.0711,3.0711,0,0,1,.65625-.84375,3.11764,3.11764,0,0,1,.91406-.57031A2.78243,2.78243,0,0,1,6.0153,6Zm-20,2A1.36869,1.36869,0,0,0,5.32,8.19531a.718.718,0,0,0-.36719.60156l-1,12.46875v.04688a.39983.39983,0,0,0,.11719.28125,1.1769,1.1769,0,0,0,.28125.21875,1.39989,1.39989,0,0,0,.34375.14063A1.28754,1.28754,0,0,0,5.0153,22h22a1.61305,1.61305,0,0,0,.32813-.03906,1.3102,1.3102,0,0,0,.35156-.125.86492.86492,0,0,0,.27344-.21875.48986.48986,0,0,0,.10938-.32031v-.09375l-1-12.32812a.83749.83749,0,0,0-.34375-.64844A1.196,1.196,0,0,0,26.0153,8ZM8.34342,24l-.3125,1.21875a5.23166,5.23166,0,0,0,.96094.25781q.6167.11719,1.375.20313.75732.08643,1.58594.14844.82764.063,1.60938.10156.78076.03955,1.44531.05469.66357.01611,1.08594.01563.84375,0,1.875-.02344t2.08594-.09375q1.05469-.07031,2.0625-.21094a12.5954,12.5954,0,0,0,1.80469-.375.12879.12879,0,0,0,.07813-.04687L23.68717,24Z" />
```
Comment (layershifter) [inline packages/fluentui/react-icons-northstar/src/components/DeviceAudioIcon.tsx:7]: What about filled/outline variations?

File: packages/fluentui/react-icons-northstar/src/components/DeviceAudioIcon.tsx [hunk 8]
```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { createSvgIcon } from '../utils/createSvgIcon';
+
+export const DeviceAudioIcon = createSvgIcon({
```
Comment (layershifter) [inline packages/fluentui/react-icons-northstar/src/components/DeviceAudioIcon.tsx:4]: I don't think that it's best name for this but I don't have ideas...

image

File: packages/fluentui/react-icons-northstar/src/components/FilesErrorIcon.tsx [hunk 9]
```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { createSvgIcon } from '../utils/createSvgIcon';
+import { iconClassNames } from '../utils/iconClassNames';
+
+export const FilesErrorIcon = createSvgIcon({
+  svg: ({ classes }) => (
+    <svg viewBox="0 0 32 32" role="presentation" className={classes.svg}>
+      <path
+        className={cx(iconClassNames.filled, classes.filledPart)}
```
Comment (layershifter) [inline packages/fluentui/react-icons-northstar/src/components/FilesErrorIcon.tsx:10]: What about outline version?

File: packages/fluentui/react-icons-northstar/src/components/FluentOneDriveIcon.tsx [hunk 10]
```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { createSvgIcon } from '../utils/createSvgIcon';
+
+export const FluentOneDriveIcon = createSvgIcon({
```
Comment (layershifter) [inline packages/fluentui/react-icons-northstar/src/components/FluentOneDriveIcon.tsx:4]:
```suggestion
export const OneDriveColorIcon = createSvgIcon({
```

Should be this `OneDriveColorIcon`?

File: packages/fluentui/react-icons-northstar/src/components/SpeakerLouderIcon.tsx [hunk 11]
```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import cx from 'classnames';
+import { createSvgIcon } from '../utils/createSvgIcon';
+import { iconClassNames } from '../utils/iconClassNames';
+
+export const SpeakerLouderIcon = createSvgIcon({
```
Comment (layershifter) [inline packages/fluentui/react-icons-northstar/src/components/SpeakerLouderIcon.tsx:6]: Should be this just `Speaker` icon?

File: packages/fluentui/react-icons-northstar/src/components/FluentOneDriveIcon.tsx (hunk 10 above)
Comment (TanelVari) [reply packages/fluentui/react-icons-northstar/src/components/FluentOneDriveIcon.tsx:4]: looks like duplicate

File: packages/fluentui/react-icons-northstar/src/components/SpeakerLouderIcon.tsx (hunk 11 above)
Comment (TanelVari) [reply packages/fluentui/react-icons-northstar/src/components/SpeakerLouderIcon.tsx:6]: Exists in FluentUI/Stardust as VolumeUpIcon

File: packages/fluentui/react-icons-northstar/src/components/CallTransferredIcon.tsx (hunk 3 above)
Comment (TanelVari) [reply packages/fluentui/react-icons-northstar/src/components/CallTransferredIcon.tsx:6]: Does not go into icons. Remains as an Illustration in TMP

File: packages/fluentui/react-icons-northstar/src/components/BreakoutRoomIcon.tsx (hunk 1 above)
Comment (TanelVari) [reply packages/fluentui/react-icons-northstar/src/components/BreakoutRoomIcon.tsx:6]: Name is fine

File: packages/fluentui/react-icons-northstar/src/components/BreakoutRoomIcon.tsx (hunk 2 above)
Comment (TanelVari) [reply packages/fluentui/react-icons-northstar/src/components/BreakoutRoomIcon.tsx]: Updated

**Review by pompomon — APPROVED** [review]

**Pull request merged**